data/config-schema/src/main/resources/net/corda/schema/configuration/messaging/1.0/corda.messaging.json

```diff
@@ -99,6 +99,13 @@
       "description": "Settings for publishers that write messages to the underlying bus",
       "$ref": "publisher.json",
       "default": {}
+    },
+    "maxAllowedMessageSize": {
```

Comment:
Does still apply to all messages from all producers? or just for messages that are being chunked. The name or description don't make that explicit if its just chunk size for large messages.

Reply:
It's for all messages. Basically, it says never publish a message larger than ....

Currently we have 2 usecases: chunking and we check the AMQP serialsed objects before publishing them to Kafka after a DB query.

Reply:
Right, it just the minimum didn't make a lot of sense to me for the case of a normal publish of one simple avro record. Do we know a single avro record can't be below 500b? If it was, will the min just be ignored and we send anyway? So min would apply to chunks but max applies to all messages.

Reply:
The minimum here is just the json schema validation. It's the minimum you're allowed to configure. So basically, we're saying you're not allowed to configure the maximum to be less than 500kb.